addons/mail/static/tests/message/message.test.js

```diff
@@ -1788,3 +1788,41 @@ test("chatter - font size unchanged when there is only emoji", async () => {
         parseFloat(getComputedStyle(textMessage).getPropertyValue("font-size"))
     );
 });
+
+test("Delete starred message updates counter", async () => {
```

Comment:
`[...] updates counter` => `[...] decrements starred counter once`

This change of wording let us understand very easily the intent of test just by its name, preventing waste of time to necessarily read the content of test in case it fails or doesn't work as expected. This looks minor now but it actually contributes quite a lot to easing maintenance of tests.